.travis-ci.sh

```diff
@@ -1,5 +1,5 @@
 # Edit this for your own project dependencies
-OPAM_DEPENDS="ocamlfind core ounit react ppx_deriving"
+OPAM_DEPENDS="ocamlfind core ounit qcheck react ppx_deriving"
```

Comment:
Change core to base.

Reply:
.travis-ci.sh is not specific to this exercise.

Reply:
Let's leave it then.